server/cmwell-bg/src/main/scala/cmwell/bg/ImpStream.scala

```diff
@@ -370,7 +360,7 @@ class ImpStream(partition: Int,
                           infoton.path.getBytes,
                           payload
               )
-            }(breakOut3))
+            }.to(IIterable))
```

Comment:
Isn't there too many I's ?

Reply:
No, it's an alias for immutable.Iterable. See the imports at the beginning of the file.